capstone/src/main/webapp/loginAuth.js

```diff
@@ -24,3 +24,10 @@ window.addEventListener('DOMContentLoaded', (event) => {
     fetchLoginStatus();
   }
 });
+
+/**
+ Only shows button after title animation finished
+ */
+setTimeout(function() {
```

Comment:
I just discovered that there's an onAnimationEnd event:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/animationend_event

Here it's probably not worth the trouble, but if you're every working on something with lots of complex animations, it could be useful to know about.